src/hooks/Timeline.js

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import { VideoContext } from '../shared/VideoContext'
+
+const baseClass = 'timeline'
+
+export const Timeline = props => {
+  const { duration, currentTime } = React.useContext(VideoContext)
+  const percentComplete = Math.floor((currentTime / duration) * 100)
+  const onMouseMove = evt => {
+    const mouseWidth = evt.pageX
+    const elementWidth = evt.target.clientWidth
+    const percentWidth = mouseWidth / elementWidth
+    const startAt = Math.floor(duration * percentWidth)
+
+    return props.onMouseMove(startAt, mouseWidth - 100)
```

Comment:
Doesn't matter, but what's the 100 for? If that's some constant width of something, can we give it a name? Or a quick comment or something?